### ISSUE TITLE
Support multiple fumble tables

### ISSUE THREAD

**Issue by mooped** [foundryvtt-dcc/dcc#269]
E.g. monster fumbles from the yearbook

**Comment by Regina-Hart** [comment]
Agreed, I would be keen to see fields for 'Fumble Die' and 'Fumble Table' comparable to what we have for 'Crit Die' and 'Crit Table.'

**Comment by cyface** [comment]
Yearbook fumble tables were implemented in a separate issue, since they have a complex implementation that runs in parallel to the crit table for monsters.
